ydb/docs/en/core/changelog-server.md

```diff
@@ -1,5 +1,85 @@
+* [Implemented](https://github.com/ydb-platform/ydb/pull/12689) a mechanism to abort volatile transactions with the ABORTED status when datashards are gracefully restarted.
+* [Added](https://github.com/ydb-platform/ydb/pull/6342) the ability to remove NOT NULL constraints on a column in a table using the `ALTER TABLE _ ALTER COLUMN _ DROP NOT NULL` query.
+* [Added](https://github.com/ydb-platform/ydb/pull/9168) the 100.000 limit on the number of simultaneous requests to create sessions in the coordination service.
+* [Increased](https://github.com/ydb-platform/ydb/pull/14219) the maximum [number of columns in the primary key](./concepts/limits-ydb.md?#schema-object) from 20 to 30.
+* **_(Experimental)_** [Added](https://github.com/ydb-platform/ydb/pull/14075) strict access control checks, which are enabled by setting the `enable_strict_acl_check` flag and the following ones:
+  * `enable_strict_user_management` enables strict checks for local users.
+  * `enable_database_admin` enables database administrator functions.
+  * `enable_data_erasure` [enables](https://github.com/ydb-platform/ydb/pull/14460) a procedure that repeatedly overwrites deleted data to minimize the risk of recovering the deleted information when directly accessing block devices via operating system functions.
+
+#### Backward incompatible changes
+
+* If you are using [temporal over YDB](https://github.com/yandex/temporal-over-ydb), update it to version [v1.23.0-ydb-compat](https://github.com/yandex/temporal-over-ydb/releases/tag/v1.23.0-ydb-compat) before updating YDB to the current version to avoid errors in query execution.
+
+#### Performance
+
+* [Added](https://github.com/ydb-platform/ydb/pull/6509) support for [constant folding](https://en.wikipedia.org/wiki/Constant_folding) in the query optimizer by default, which improves query performance by evaluating constant expressions at compilation time. This feature reduces runtime overhead, enabling faster and more efficient query execution for complex static expressions.
+* [Added](https://github.com/ydb-platform/ydb/issues/6512) a granular timecast protocol that will reduce the execution time of distributed transactions (slowing down one shard will not cause all shards to slow down).
+* [Implemented](https://github.com/ydb-platform/ydb/issues/11561) in-memory state migration on graceful restart, which allows preserving locks and increasing the chances of successful transaction execution. This reduces the execution time of long transactions by decreasing the number of retries.
+* [Optimized](https://github.com/ydb-platform/ydb/pull/15264) memory consumption by storage nodes.
+* [Improved](https://github.com/ydb-platform/ydb/pull/10969) Hive startup times.
+* [Optimized](https://github.com/ydb-platform/ydb/pull/6561) the replication process.
+* [Optimized](https://github.com/ydb-platform/ydb/pull/9491) the header size of large binary objects in VDisk.
+* Improved diagnostics and introspection of memory errors.
```

Comment:
Diagnostics and introspections sounds like a feature, not a performance improvement. Missing documentation link.